picard/ui/options/plugins.py

```diff
@@ -159,7 +158,19 @@ def _populate(self):
 
         self._user_interaction(True)
 
+    def _remove_all(self):
+        self._user_interaction(False)
+        for i, p in self.items.items():
+            idx = self.ui.plugins.indexOfTopLevelItem(i)
+            item = self.ui.plugins.takeTopLevelItem(idx)
+            del item
```

Comment:
I'm not entirely sure why you're calling `del item` here, but not, for example, `del idx`. In general, you don't need to call `del` on small variables that go out of scope in a few lines.